app/views/offline_cancellation_mailer/send_notification.html.haml

```diff
@@ -1,4 +1,6 @@
--# TODO: content needs fleshing out
-
-Because #{@instrument} is unavailable, your reservation for
-#{@reservation.reserve_start_at} has been canceled.
+%p
+  = html "body.html",
+    facility_url: facility_url(@reservation.facility),
+    product: @reservation.product,
+    reservation_time: I18n.l(@reservation.reserve_start_at, format: :timeonly),
+    reservation_url: order_order_detail_reservation_url(@reservation.order, @reservation.order_detail, @reservation)
```

Comment:
I'm thinking the order detail link might be a little more useful. It includes the time as well as the cost and other information. The reservation view doesn't have a link back to the order detail (which it probably should at some point)